lib/process-services-cloud/src/lib/group/components/group-cloud.component.scss

```diff
@@ -14,6 +14,10 @@
         &-cloud-group {
             width: 100%;
 
+            &-option-not-active {
+                background: #ffffff !important;
```

Comment:
you need to get the color from the theme

Reply:
Can you also write the message just in nromal text color,  because is not an error and remove the icon, thanks